website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -26,6 +26,24 @@ to `true`. In prior Consul versions (1.10.x through 1.11.x), the config defaulte
 well as the flag will be removed in upcoming Consul 1.13. We recommend changing your instrumentation to use 1.10 and later
 style `consul.api.http...` metrics and removing the configuration flag from your setup.
 
+### TLS Configuration
+
+TLS can now be configured differently on each of Consul's exposed listeners/ports.
+As such, the following top-level configuration fields are deprecated and should be
+replaced with the new `tls` stanza:
```

Comment:
I'd like to link to the new TLS docs from here, but it looks like they were accidentally deleted in https://github.com/hashicorp/consul/commit/f4c03d2340d5650ba409aa3f4bfa813560d165b2 so will wait until they're restored 🙇🏻‍♂️